src/Illuminate/Database/Connectors/SqlServerConnector.php

```diff
@@ -126,6 +126,10 @@ protected function getSqlSrvDsn(array $config)
             $arguments['TrustServerCertificate'] = $config['trust_server_certificate'];
         }
 
+        if (isset($config['multiple_active_resultsets']) && $config['multiple_active_resultsets'] === false) {
```

Comment:
Hi, thanks for your observation, Laravel 5.4 still supports PHP 5.6 which does not have the null coalesce operator, so, unfortunately that cannot be used. I hope it gets merged on Laravel 5.4 because that is what I use in production.

Also the argument should only be set to the string value `'false'` (not the `false` boolean value) only when explicitly disabled, because:
1. MARS is enabled by default (see the [Microsoft docs](https://github.com/Microsoft/msphpsql/wiki/Recommendations-for-improving-the-performance-of-PDO_SQLSRV-and-SQLSRV#1-multiple-active-result-sets-mars) linked above)
2. The [`buildConnectString(...)`](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Connectors/SqlServerConnector.php#L139-L144) method uses `sprintf` so a `false` boolean value would be outputted as the string `'0'`, and if you look in Microsoft documentation linked above the added option to the connectiion string should be `MultipleActiveResultSets=false`
3. I used the [`$config['pooling']`](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Connectors/SqlServerConnector.php#L113-L115) (a few lines above) as an example on how to do it.